applications_tests/lethe-particles/insert_file_3d.prm

```diff
@@ -79,7 +79,7 @@ end
 
 subsection insertion info
   set insertion method    = file
-  set insertion file name = ../particles.input
+  set list of input files = ../particles.input
```

Comment:
I feel like the parameter can be called "input file list".

Reply:
I can, but I don't think it should.

Reply:
this is correct right now I think